src/lib.rs

```diff
@@ -693,6 +693,79 @@ pub fn format_input<T: Write>(
     }
 }
 
+/// A single span of changed lines, with 0 or more removed lines
+/// and a vector of 0 or more inserted lines.
+#[derive(Debug, PartialEq, Eq)]
+pub struct ModifiedChunk {
+    /// The first affected line before formatting.
+    pub line_number: u32,
```

Comment:
To correspond with `Mismatch`, this should be `line_number_orig`, the comment might be better phrased as "The first line to be removed from the original text"

Reply:
That sounds like better wording, thanks!